source/js/foundation/template-js-handler/custom-hero-video-handler.js

```diff
@@ -1,8 +1,8 @@
 /**
- * Adding play/pause functionality to the optional Publication/Article Page hero video section.
+ * Adding play/pause functionality to the optional custom hero video.
  */
 
-export default function publicationArticleHeroVideoHandler() {
+export default function customHeroHeroVideoHandler() {
```

Comment:
**Typo** There is "hero" in there twice. I believe that would break the usages of the function.